legal-api/migrations/versions/bf8d4391745e_cleanup_document_table.py

```diff
@@ -0,0 +1,33 @@
+"""cleanup-document-table
+
+Revision ID: bf8d4391745e
+Revises: bddd732a9269
+Create Date: 2023-01-03 17:42:27.696030
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = 'bf8d4391745e'
+down_revision = 'bddd732a9269'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('documents', 'content_type')
+    op.drop_column('documents', 'file_name')
+    op.drop_column('documents_version', 'content_type')
+    op.drop_column('documents_version', 'file_name')
+    # ### end Alembic commands ###
```

Comment:
so i guess the idea is only the document key and document type are needed at the end of the day?

Reply:
yes. since we are not using file_name and content_type field, removing it